pkgs/tools/wayland/wdomirror/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, meson
+, ninja
+, pkg-config
+, wayland
+, wayland-protocols
+}:
+
+stdenv.mkDerivation {
+  pname = "wdomirror";
+  version = "unstable-2021-01-08";
+
+  src = fetchFromGitHub {
+    owner = "progandy";
+    repo = "wdomirror";
+    rev = "e4a4934e6f739909fbf346cbc001c72690b5c906";
+    sha256 = "1fz0sajhdjqas3l6mpik8w1k15wbv65hgh9r9vdgfqvw5l6cx7jv";
+  };
+
+  nativeBuildInputs = [ meson ninja pkg-config wayland wayland-protocols ];
```

Comment:
Please move the libraries that are linked to buildInputs.

Reply:
I'm moving wayland to buildInputs, as it is definitely linked.

From my understanding wayland-protocols is used to generate sources, which are then compiled and linked into the final executable, so I don't think it needs to be in buildInputs. This may also solve the question in [wl-clipboard](https://github.com/NixOS/nixpkgs/blob/57231d7524d94dd992bb8726af8c941d11e2bee3/pkgs/tools/wayland/wl-clipboard/default.nix#L33).

Reply:
Yup, looks good to me: wayland-scanner is in wayland-protocols